app/Console/Commands/CleanAcarsMessages.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Acars\AcarsMessage;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class CleanAcarsMessages extends Command
+{
+    protected $signature = 'acars:clean-history';
+
+    protected $description = 'Delete any acars message history that is older than a specified age';
```

Comment:
"Than a specified age" but then specifies to subtract a month from the current date. Which one did you intend? :D

Reply:
Haha one month, changed